components/chainhook-cli/src/config/generator.rs

```diff
@@ -36,6 +39,8 @@ max_number_of_processing_threads = 16
 max_number_of_networking_threads = 16
 max_caching_memory_size_mb = 32000
 
+# The TSV file is required for downloading historical data for your predicates. If this is not a requirement, you can comment out the `tsv_file_url` line.
+#
```

Comment:
Sorry for the nit! But this line was pretty long and we had an extra `#` at the end.
```suggestion
# The TSV file is required for downloading historical data for your predicates. 
# If this is not a requirement, you can comment out the `tsv_file_url` line.
```